### DISCUSSION TITLE
.NET MAUI - How to use Tablet running windows OS as debugging device

### DISCUSSION THREAD

**Discussion by dmvaldez09** [dotnet/maui#15083] [Q&A]
I need some help.

If an external android device can be configured as debugging for android platform, how about windows platform?

How do I setup my tablet running windows 10 as an external device for debugging?

I opened a question in stackoverflow but was closed and downvoted.

There were no mentioning of this in the docs.

**Comment by mkenith99** [comment]
I am having the same issue. I was wondering how to enable live debugging on an actual windows tablet.

**Comment by dmvaldez09** [comment]
@drasticactions would you care to answer please?

**Reply by drasticactions** [reply]
https://learn.microsoft.com/en-us/visualstudio/debugger/remote-debugging?view=vs-2022&preserve-view=true

Using the Windows Remote Debugging tools.
